website/docs/understanding_ferretdb.md

```diff
@@ -56,6 +56,69 @@ For complex documents, you can nest objects (subdocuments) inside a document.
 
 In the example above, the `name` field is a subdocument embedded into a document.
 
+## Dot Notation
+
+Dot notations `(.)` are used to specify the elements in an array, the objects in a document, or the values in an embedded document.
+
+### Arrays
+
+Dot notations can be used to specify or query the elements present in an array by concatenating a dot `(.)` with the index position of the element.
+
+```sh
+"array_name.index"
+```
+
+:::note
+When using dot notations, the field name of the array and the specified element must be enclosed in quotation marks.
+:::
+
+For example, let's take the following array field in a document:
+
+```sh
+animals: ["dog", "cat", "fish", "fox"]
+```
+
+To reference the fourth element in the array, use the dot notation `" animals.4"`.
```

Comment:
```suggestion
To reference the fourth element in the array, use the dot notation `"animals.3"`.
```
Could you double check if index starts start 0 or 1? previous example made me think index starts at 0.